nautilus_trader/adapters/interactive_brokers/client/market_data.py

```diff
@@ -418,6 +419,7 @@ async def subscribe_historical_bars(
         contract: IBContract,
         use_rth: bool,
         handle_revised_bars: bool,
+        start: int | None = None,
```

Comment:
Consider making this a `pd.Timestamp` and we can always call `.value` later? (otherwise would be standard to use `start_ns`)